android/gltfio-android/src/main/java/com/google/android/filament/gltfio/FilamentInstance.java

```diff
@@ -66,6 +67,20 @@ void clearNativeObject() {
         return result;
     }
 
+    /**
+     * Gets the <code>NameComponentManager</code> label for the given entity, if it exists.
+     */
+    public String getName(@Entity int entity) {
```

Comment:
Can't  users just do `instance->getOwner()->getName(..)`?

Reply:
There's no `getOwner()` in Android